src/web/app/src/components/DependenciesPage.tsx

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from '@material-ui/core/styles';
+import useSWR from 'swr';
+
+import DependenciesTable from './DependenciesTable';
+import { dependencyDiscoveryUrl } from '../config';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    fontFamily: 'Spartan',
+    padding: '1em 0 2em 0',
+    paddingTop: 'env(safe-area-inset-top)',
+    wordWrap: 'break-word',
+    [theme.breakpoints.down(1024)]: {
+      maxWidth: 'none',
+    },
+    '& h1': {
+      color: theme.palette.text.secondary,
+      fontSize: 24,
+      transition: 'color 1s',
+      marginTop: 0,
+    },
+    '& p, blockquote': {
+      color: theme.palette.text.primary,
+      fontSize: 16,
+      padding: 8,
+    },
+  },
+  container: {
+    padding: '2vh 18vw',
+    [theme.breakpoints.down(1024)]: {
+      padding: '2vh 8vw',
+      wordWrap: 'break-word',
+    },
+  },
+}));
+
+const DependenciesPage = () => {
+  const classes = useStyles();
+  const { data: dependencies = [] } = useSWR(
+    `${dependencyDiscoveryUrl}/projects`,
+    async (u: string) => {
+      const fetchDependenciesReq = await fetch(u);
+      const fetchDependenciesJson = await fetchDependenciesReq.json();
+      return fetchDependenciesJson;
+    }
+  );
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.container}>
+        <h1>Dependencies</h1>
+        <p>
+          Telescope is an open source project, built with other open source projects. We use more
+          than 1,000 libraries, frameworks, tools and other projects, and each one has opportunities
+          for you to contribute fixes, features, documentation, translations, and tests. Find your
+          next Issue in the list below, or search for for something familiar. Help improve Telescope
```

Comment:
```suggestion
          next Issue in the list below, or search for something familiar. Help improve Telescope
```
oops

Reply:
Also, the text seems lengthy on mobile and covers 1/2 of the screen.

Reply:
Fixed